### DISCUSSION TITLE
Welcome to brick discussions!

### DISCUSSION THREAD

**Discussion by jtdaugherty** [jtdaugherty/brick#425] [General]
## Welcome!
  This discussion area is a place to connect with other `brick` users and get help. We hope that you:
  * Ask questions, particularly things that don't seem like issues or problems. This could include stuff like application design questions, API usage, etc.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by levans** [comment]


**Reply by jtdaugherty** [reply]
Hi @levans - would you be willing to open a new discussion item with this? Once you've done that, I'll remove this since this item is just a welcome placeholder. Thanks!

**Comment by levans** [comment]
Oops, sorry, I wasn’t logged into GitHub when I went to the Discussions section and failed to see all the topics.

Will post in the right place!
